### PR TITLE
DM-25518: Update scripts to use ``amain``.

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: scripts/auxtel/enable_atcs.py [hunk 1]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/enable_atcs.py:4]: Telescope and Site.

File: scripts/auxtel/enable_latiss.py [hunk 2]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/enable_latiss.py:4]: Telescope and Site

File: scripts/auxtel/prepare_for_flat.py [hunk 3]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/prepare_for_flat.py:4]: Telescope and Site

File: scripts/auxtel/prepare_for_onsky.py [hunk 4]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/prepare_for_onsky.py:4]: Telescope and Site

File: scripts/auxtel/shutdown.py [hunk 5]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/shutdown.py:4]: Telescope and Site

File: scripts/auxtel/standby_latiss.py [hunk 6]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/standby_latiss.py:4]: Telescope and Site

**Comment by r-owen** [comment]
Good catch. I did a global search and replace to fix all the incorrect references to Data Management.

**Pull request merged**